exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if(sentence.includes("code")
+  ){
```

Comment:
We want to keep the `if` parenthesis in one line, with no new line. So line 13 goes to the end of line 12 here